### DISCUSSION TITLE
PyConES 2023 talk featuring Pyodide

### DISCUSSION THREAD

**Discussion by simonegiacomelli** [pyodide/pyodide#4114] [Show and tell]
Hello everyone!

I'm thrilled to share that my proposal for PyConES 2023 has been accepted! 🎉

The talk, titled "Python all the way down: Seamless Server and in-Browser web programming with wwwpy, Pyodide, and WASM." aims to explore the pivotal role of Pyodide in modern web programming. I hope the presentation brings more recognition and admiration to the fantastic work done here.

You can check out the abstract of the talk [here](https://charlas.2023.es.pycon.org/pycones-2023/talk/review/YJRGRZLFAVVRPBA7WVUKPPBBVLHXNMDY).

I'm genuinely grateful to this community for providing invaluable insights and resources. As I've worked on wwwpy (currently in its third complete rewrite, by the way 😅), I've often sent my thanks and good vibes to the Pyodide project. In fact, wwwpy wouldn't exist without Pyodide (obviously!).

I'd love to hear any feedback or suggestions you might have before the actual presentation!

**Comment by ryanking13** [comment]
Very cool! I look forward to watching the presentation :)
